workflows/checks/check-md5.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// run parameters
+params.run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv'
+params.outdir = 's3://nextflow-ccdl-results/scpca-checks/'
+
```

Comment:
You did not define the params like you did in the other scripts (i.e `params.run_ids = "SCPCR000001,SCPCR000002"`), are they being passed into this script somewhere? Why are they not needed here?